batch/vep/build_vep_cache.sh

```diff
@@ -47,19 +47,19 @@ readonly ftp_base="ftp://ftp.ensembl.org/pub/release-${release}"
 # Also the first letter of the file is capital while it is small for the actual
 # cache file (above). For example: "Homo_sapiens.GRCh38.dna.toplevel.fa.gz"
 readonly fasta_file="${species^?}.${assembly}.dna.toplevel.fa.gz"
-if [ $species == "homo_sapiens" ] && [ $assembly == "GRCh37" ]; then
-  if [ ! `command -v samtools` ]; then
+if [[ $species == "homo_sapiens" ]] && [[ $assembly == "GRCh37" ]]; then
+  if [[ ! `command -v samtools` ]]; then
     echo "ERROR: samtools is needed to create the .fai index."
-    echo -e "It can be installed by:\n"
-    echo -e "sudo apt-get install samtools\n"
+    echo -e "It can be installed by:"
```

Comment:
Please drop `-e` from here and the rest of echo commands (since you don't have `\n` anymore).

Reply:
Nice catch; thank you!